doc/run/repl.md

```diff
@@ -1,16 +1,18 @@
-# Run SQLFlow REPL
+# Run SQLFlow command-line tool
 
-In addition to building SQLFlow into a gRPC server, accessed via Jupyter Notebook, we could also build it into a command line program, whose `main` function reads SQL statements from the console, evaluates them by calling SQLFlow, and prints the results.  This command-line program makes it easy to debug and profile locally without starting the SQLFlow server and the Jupyter server.  We call this command-line program the SQLFlow REPL. SQLFlow REPL supports automatic code completion to ease applying the underlying powerful AI toolset of SQLFlow.
+We have a command-line tool called `sqlflow` which can access SQLFlow server like we do in Jupyter Notebook.  `sqlflow` works in the well known `REPL` mode, that is to say, reads SQL statements from the console, evaluates them by calling SQLFlow server, and prints the results.  `sqlflow` supports automatic code completion to ease applying the underlying powerful AI toolset of SQLFlow.
 
 ![](figures/repl.gif)
 
 ## Quick Start
 
-The SQLFlow Docker image contains the REPL command-line program.  We can run an example session by typing the following command on MacOS.  If you run Docker on Linux, please change `host.docker.internal:3306` to `localhost:3306`.
+The SQLFlow Docker image contains the `sqlflow` client.  We can run an example session by typing the following command on MacOS.  If you run Docker on Linux, please change `host.docker.internal:3306` to `localhost:3306`.
```

Comment:
Why do we need to include the client tool in the Docker image? I think the image could contain only the gRPC server and its dependencies.  Users run the client from their native computing environment like macOS, Linux, and Windows.

Reply:
> Why do we need to include the client tool in the Docker image? I think the image could contain only the gRPC server and its dependencies. Users run the client from their native computing environment like macOS, Linux, and Windows.

Agreed.

Reply:
done